openpype/hosts/unreal/api/plugin.py

```diff
@@ -1,7 +1,251 @@
 # -*- coding: utf-8 -*-
-from abc import ABC
+import sys
+import six
+from abc import (
+    ABC,
+    ABCMeta,
+)
 
-from openpype.pipeline import LoaderPlugin
+import unreal
+
+from .pipeline import (
+    create_publish_instance,
+    imprint,
+    lsinst,
+    UNREAL_VERSION
+)
+from openpype.lib import (
+    BoolDef,
+    UILabelDef
+)
+from openpype.pipeline import (
+    Creator,
+    LoaderPlugin,
+    CreatorError,
+    CreatedInstance
+)
+
+
+class OpenPypeCreatorError(CreatorError):
```

Comment:
Why to create additional error? (with `OpenPype`)

Reply:
I've followed Houdini and Nuke as base, and they both do it 👀

Reply:
:see_no_evil::hear_no_evil::speak_no_evil:

Reply:
It's not needed to have custom exception. Use `CreatorError` as is.